ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -187,6 +190,11 @@ export default class ConfirmPageContainerContent extends Component {
         {ethGasPriceWarning && (
           <ConfirmPageContainerWarning warning={ethGasPriceWarning} />
         )}
+        {txData?.securityProviderResponse?.flagAsDangerous !== 0 && (
+          <SecurityProviderBannerMessage
+            securityProviderResponse={txData?.securityProviderResponse}
+          />
+        )}
         <ConfirmPageContainerSummary
           className={classnames({
             'confirm-page-container-summary--border':
```

Comment:
Will be nice to have test coverage of the code change in this component.

Reply:
Fixed here: 40ce40fc9a05ea53d8c76e6b10af5a09783e4b00